examples/es6/src/index.ts

```diff
@@ -55,8 +55,18 @@ const returnOK = <T extends ResponseData>(res: Response, out: SdkResponse<T>) =>
 };
 
 const returnCookies = <T extends ResponseData>(res: Response, out: SdkResponse<T>) => {
-  if (out.ok && out.data?.cookies) {
-    res.set('Set-Cookie', out.data.cookies);
+  if (out.ok) {
```

Comment:
Can you please elaborate why this change is needed?
AFAIR the data.cookies already contains both session & refresh tokens

Reply:
I think this is something that should be done on the node sdk side, and not in the example app
We already taking the refresh token from the body, why not taking the session as well?
https://github.com/descope/node-sdk/blob/fba980344a86a7581eceee0367028cbb7d7301dd/lib/helpers.ts#L45

Reply:
Another question, what if it's too big to store in a cookie?
Do we need to check it?

Reply:
https://github.com/descope/etc/issues/1258